server/opts.go

```diff
@@ -27,13 +27,27 @@ import (
 	"regexp"
 	"strconv"
 	"strings"
+	"sync/atomic"
 	"time"
 
 	"github.com/nats-io/jwt"
 	"github.com/nats-io/nats-server/conf"
 	"github.com/nats-io/nkeys"
 )
 
+var allowUnknownTopLevelField = int32(0)
```

Comment:
Only top level?

Reply:
That's the one I need (otherwise would also need to go through all places with `default` and ignore error, but I really don't need that).